src/components/Table/index.js

```diff
@@ -1,21 +1,76 @@
 // import {useState} from 'react';
-import StyledRow from './StyledRow';
-import StyledRowItems from './StyledRowItems';
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+import styled from 'styled-components';
 
+import mockData from '../lib/mock-data.json';
+import RunDataForm from '../RunDataForm';
+/**
+ * ##################### STYLING #####################
+ */
+const StyledTableContainer = styled.table`
+	width: 90vw;
+	margin: 50px auto;
+	border: 2px solid black;
+`;
+const StyledTRHead = styled.tr`
+	background-color: #cce1f5;
+`;
+const StyledTRBody = styled.tr`
+	background-color: #ddebf8;
+`;
+const StyledTH = styled.th`
+	padding: 5px 0;
+`;
+const StyledTD = styled.td`
+	padding: 5px 0;
+	text-align: center;
+`;
+/**
+ * ##################### COMPONENT #####################
+ */
 export default function DataTable() {
+	const [runData, setRunData] = useState(mockData);
+
+	function addRunningData(event) {
+		console.log('2');
+		console.log('OUTside IF: ' + event.distance + '\n');
+
+		if (event.distance.substring(',')) {
+			console.log('INside IF: ' + event.distance);
+			console.log('This one has a comma!' + '\n');
+
+			console.log(event.distance.toString());
+			// event.distance.replace(',' , '.');
+		}
```

Comment:
I wonder, of course, what you need those fancy `console.log()` for. 
Clean Code is King - we want to keep our code as clean as possible.  
But in this special case, I'll turn a blind eye because the rest of your code and your structure is super duper clean. 👑

Reply:
@JessicaLoers 
The only and really only reason for those `console.logs`still being there is, because I overlooked them. I promise. 🤞🏼